third_party/onnx.BUILD

```diff
@@ -43,19 +43,7 @@ genrule(
 )
 
 cc_library(
-    name = "onnx",
-    srcs = glob(
-        [
-            "onnx/*.cc",
-            "onnx/common/*.cc",
-            "onnx/defs/**/*.cc",
-            "onnx/shape_inference/*.cc",
-            "onnx/version_converter/*.cc",
-        ],
-        exclude = [
-            "onnx/cpp2py_export.cc",
-        ],
-    ),
+    name = "onnx_headers",
```

Comment:
Separated out the headers is required since onxx contain static globals, which should't be statically linked into multiple shared libraries.